src/ApiService/ApiService/onefuzzlib/notifications/NotificationsBase.cs

```diff
@@ -93,13 +99,15 @@ protected class Renderer {
             _reportUrl = reportUrl;
             _targetUrl = targetUrl;
             _inputUrl = inputUrl;
+            _shouldConvertJinja = shouldConvertJinja || true;
```

Comment:
`_shouldConvertJinja` will always be `true`?

Reply:
The default for the FF if it doesn't exist is false and I wanted it to be enabled when we deployed.

I renamed the FF to `scribanOnly` so that it's compatible with the default false.